.github/workflows/ci.yaml

```diff
@@ -36,7 +36,11 @@ jobs:
         node-version: '15'
 
     - name: Build wasm
-      run: wasm-pack build --target web
+      run: |
+        cd rust-pack
+        cargo run
```

Comment:
Would compiling in release mode be a net win in time it takes to compile and run combined?

Reply:
I didn't see difference. It is fast (=<5 seconds relative to optimizing wasm which took one minute minimum) so it doesn't matter?